tests/unit/retries/test_standard.py

```diff
@@ -26,7 +26,6 @@
     (400, 'LimitExceededException', True),
     (403, 'RequestThrottled', True),
     (503, 'SlowDown', True),
-    (503, 'SlowDown', True),
```

Comment:
I think we're actually missing an item in the list. It looks like we should have this line in its place:

```
(400, 'TransactionInProgressException', True),
```

Can we add that on line 24 while we're here?

Reply:
Sure just added that to line 24.